Header header
float64 data # Represents bpm for rr and hr nodes